docs/tutorials/maps.ipynb

```diff
@@ -779,18 +1036,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "Table.read(hdulist[\"PRIMARY_BANDS\"])"
+    "iem_plus_iem = m_iem_10GeV + m_iem_10GeV\n",
+    "\n",
+    "iem_minus_iem = m_iem_10GeV - m_iem_10GeV\n"
```

Comment:
maybe show multiplication as well, and that in Gammapy, multiplication is not commutative - ie, `map * int` works but not `int * map` not.